dependencies.rb

```diff
@@ -0,0 +1,40 @@
+require 'yaml'
+require 'pry'
+require 'i18n'
+
+I18n.load_path << Dir["#{File.expand_path('config/locales', __dir__)}/*.yml"]
+I18n.default_locale = :en
```

Comment:
It's better to move this config lines to separate file (like ./config/i18n.rb)

Reply:
thanks